src/main/java/com/example/chat/groupchatbackend/controllers/UserController.java

```diff
@@ -36,6 +45,59 @@ public ResponseEntity register(@RequestBody User user) {
                 .body("User already exists");
     }
 
+    @GetMapping("/messages/{conversationName}?newerThan={timestamp}")
```

Comment:
should be `@GetMapping("/messages/{conversationName}")` and instead of @PathVariable for timestamp, there should be @RequestParam used